apps/condo/domains/ticket/schema/TicketChange.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Generated by `createschema ticket.TicketChange 'ticket:Relationship:Ticket:CASCADE;'`
+ */
+
+const { Relationship } = require('@keystonejs/fields')
+const { GQLListSchema } = require('@core/keystone/schema')
+const { uuided, tracked } = require('@core/keystone/plugins')
+const { DV_FIELD } = require('@condo/domains/common/schema/fields')
+const access = require('@condo/domains/ticket/access/TicketChange')
+const { generateChangeTrackableFieldsFrom } = require('@condo/domains/common/utils/serverSchema/changeTrackable')
+const { fieldsToTrackInTicketForChanges } = require('./Ticket')
+
+/**
+ *
+ * @type {GQLListSchema}
+ */
+const TicketChange = new GQLListSchema('TicketChange', {
+    schemaDoc: 'Incremental changes of Ticket',
+    fields: {
+        dv: DV_FIELD,
+
+        ticket: {
+            schemaDoc: 'Related ticket, whose change is logged in this entity',
+            type: Relationship,
+            ref: 'Ticket',
+            isRequired: true,
+            knexOptions: { isNotNullable: true }, // Required relationship only!
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
+        },
+        ...generateChangeTrackableFieldsFrom(fieldsToTrackInTicketForChanges),
+    },
+    plugins: [uuided(), tracked()],
+    access: {
+        read: access.canReadTicketChanges,
+        create: access.canManageTicketChanges,
```

Comment:
It’s better to just return false. It’s by design :)

Reply:
I need to create `TicketChange` on a server side.
AFAIK, Keystone does not creates appropriate mutation, when `false` will be assigned to `access` object keys.
As a result, when I assign `access.create: false`, I get following error:
> GraphQLError [Object]: Variable "$data" expected value of type "TicketChangeCreateInput" which cannot be used as an input type.
> Error: [error] Create TicketChange internal error